### ISSUE TITLE
A user's OWN profile page should not have sort type settings applied to it when a user views it

### ISSUE THREAD

**Issue by dannirook** [LemmyNet/lemmy-ui#1842]
### Requirements

- [X] This is a feature request and not a bug report. Otherwise, please create a new [bug report](https://github.com/LemmyNet/lemmy-ui/issues/new?assignees=&labels=bug%2Ctriage&projects=&template=BUG_REPORT.yml) instead.
- [X] Please [check](https://github.com/LemmyNet/lemmy-ui/issues) to see if this request (or a similar one) already exists.
- [X] It's a single feature. Please don't request multiple features in one issue.

### Describe the feature you'd like

I'm unsure if this is bug of feature.  It FEELS like a bug, but I also suspect it's this way because nobody thought to program it any another way yet, which would make this a new feature?  

Either way--categorize as whichever makes sense.

* Users use their Profile page to basically moderate and edit their own posts.
* When a user visits their own profile page ( Username > Profile ), there is a filter applied to it.  This filter is the SAME one as the one used for the front page and all other pages, and is set by the user in Settings.
* **Scenario:** I, the user, think front page Sort Type as Hot is trash, want Top Six Hours as default instead.
* I change my default Sort Type under settings, believing it ONLY applies to the stuff I'm browsing and not my own profile, and Save.
* I browse the front page for 6 hours, lurking, making no posts
* I see a post I made yesterday in a small sub has a typo--I want to fix it!
* I go to my own profile.  It is utterly blank.  It is showing me my OWN posts with the "Top Six Hours" filter applied, making it appear that all my comments have vanished.
* I am not a techie user, so I freak out wondering why all my posts have vanished.  It's not logical to me that my Top Six Hours sort for the front page applies to a list of my own posts on my profile.  Other sites don't work like that.  So my posts look like they're just gone.
* Also, there is no clear button to immediately strip all filters (overriding settings) and show all your own posts chronologically.
* Solution: Don't sort a user's own profile page by their Sort Type setting in Settings.  Create another option in Settings for it, or just leave it as a chronological list of when things were posted, so the user can moderate their own posts without things seeming to vanish.

TL;DR: Users edit/interact with their own posts via Profile.  Sort Type for front page ALSO currently applies to profile, making it appear that a user's own posts have vanished if a setting like Top Six Hours was selected for front-page viewing.

FYI: I also put in a bug report for the Language filter doing basically the same thing to a user's profile.  https://github.com/LemmyNet/lemmy-ui/issues/1841 .  

My guess is these two things touch the same code--but one is language and one is Sort Type filters so I didn't want to assume, and made two different posts.
